### ISSUE TITLE
Adjustable progress update interval (for tilelive-copy)

### ISSUE THREAD

**Issue by jflemer-ndp** [mapbox/tilelive#171]
It would be nice if the progress update interval was adjustable (in `tilelive-copy`).  Progress output is really nice ... but the hard coded default of 100 ms is pretty verbose on a long copy.  I'd be happy with something in the range of 5-15 minutes for some operations (a 9000:1 decrease in log size).

https://github.com/mapbox/tilelive/blob/efdfe7d2b121ccc8ace74844088913f1f981abcc/lib/tilelive.js#L328

**Comment by rclark** [comment]
Yeah this would be a nice improvement. We implemented this in [mapbox-tile-copy](https://github.com/mapbox/mapbox-tile-copy/blob/master/bin/mapbox-tile-copy.js#L55-L59) and the same general logic could be ported back here to tilelive-copy.
